L1Trigger/L1THGCal/src/backend/HGCalHistoSeedingImpl.cc

```diff
@@ -35,11 +35,26 @@ HGCalHistoSeedingImpl::HGCalHistoSeedingImpl(const edm::ParameterSet& conf)
     throw cms::Exception("HGCTriggerParameterError")
         << "Unknown Seed Position option '" << conf.getParameter<std::string>("seed_position");
   }
+  if (conf.getParameter<std::string>("seeding_space") == "RPhi") {
+    seedingSpace_ = RPhi;
+  } else if (conf.getParameter<std::string>("seeding_space") == "XY") {
+    seedingSpace_ = XY;
+  } else {
+    throw cms::Exception("HGCTriggerParameterError")
+      << "Unknown seeding space  '" << conf.getParameter<std::string>("seeding_space");
+  }
 
   edm::LogInfo("HGCalMulticlusterParameters")
       << "\nMulticluster MIPT threshold for histo threshold algorithm: " << histoThreshold_
       << "\nMulticluster type of multiclustering algortihm: " << seedingAlgoType_;
 
+    if (seedingAlgoType_.find("Histo") != std::string::npos && seedingSpace_ == RPhi &&
+	nBins1_ != binsSumsHisto_.size()) {
+      throw cms::Exception("Inconsistent bin size")
+        << "Inconsistent nBins_X1_histo_multicluster ( " << vnBins1_[0] << " ) and binSumsHisto ( " << binsSumsHisto_.size()
+        << " ) size in HGCalMulticlustering\n";
+    }
```

Comment:
- `nBins1_` is not properly defined at this stage. 
- Inconsistent indent and tab/space mixing to be fixed